test/Magneto.Tests/Core/CachedQueryExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using System.Reflection;
+using Magneto.Core;
+
+namespace Magneto.Tests.Core
+{
+	public static class CachedQueryExtensions
+	{
+		public static string GetCacheKey<TContext, TCacheEntryOptions, TCachedResult>(this CachedQuery<TContext, TCacheEntryOptions, TCachedResult> cachedQuery)
+		{
+			return cachedQuery.getStateProperty<string>("CacheKey");
+		}
+
+		public static TCachedResult GetCachedResult<TContext, TCacheEntryOptions, TCachedResult>(this CachedQuery<TContext, TCacheEntryOptions, TCachedResult> cachedQuery)
+		{
+			return cachedQuery.getStateProperty<TCachedResult>("CachedResult");
+		}
+
+		static T getStateProperty<T>(this object cachedQuery, string name)

```

Comment:
camelCasing looks odd to me here

Reply:
This is the naming convention used in this project (private methods are camelCased).